src/common/check_dataset_schema/script.py

```diff
@@ -22,7 +24,7 @@ def check_structure(slot_info, adata_slot):
   return missing
 
 print('Load data', flush=True)
-adata = ad.read_h5ad(par['input']).copy()
+adata = ad.read_h5ad(par['input'], backed='r')
```

Comment:
Due to this change `.X` is not available anymore if it is empty/None and causes an error downstream on line 100:

scverse/anndata#715